frontend/app/(main)/page.module.scss

```
@@ -16,7 +16,11 @@
   min-height: 0;
   background-color: #ffffff;
   padding: 10px;
-  overflow-x: hidden;
+  // DailyView/WeeklyView each own their own bounded internal scroll region now, so this
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)